articles/index.yml

```
@@ -22,107 +22,112 @@ landingContent:
     linkLists:
       - linkListType: overview
         links:
-          - text: Five questions about quantum computing
+          - text: The Quantum Development Kit
             url: welcome.md
-      - linkListType: get-started
-        links:
-          - text: Build a quantum random number generator
-            url: quickstarts/qrng.md
-          - text: Quantum basics with Q#
-            url: quickstart.md            
-          - text: Learn about impactful quantum programs with Q#
-            url: quickstarts/search.md
-          - text: Learn about solving our biggest challenges in chemistry
-            url: https://cloudblogs.microsoft.com/quantum/2018/12/04/simulating-nature-with-the-new-microsoft-quantum-development-kit-chemistry-library/
       - linkListType: learn
         links:
-          - text: Learn with the Quantum Katas
-            url: intro-to-katas.md
-          - text: Browse Q# samples
-            url: https://docs.microsoft.com/samples/browse/?languages=qsharp
+          - text: What is quantum computing?
+            url: overview/what-is-quantum-computing.md
+          - text: What can a quantum computer do?
+            url: overview/quantum-computers.md
+          - text: Why learn quantum computing?
+            url: overview/why-learn-quantum-computing.md
+          - text: What is Q#?
+            url: overview/what-is-qsharp.md
+          - text: How can I learn about quantum computing?
```

```suggestion
          - text: How to learn quantum computing?
```
Keeping title consistence withing sections

---

I was actually meaning to inquire about this a while back (regarding the general section name, not just here). Definitely want to get more opinions (@cgranade @bromeg et al.) because I'm not sure if I'm being crazy, but doesn't "How to learn quantum computing?" sound very awkward phrased as a question? In that form it seems like we're just tacking on the question mark to put it in the "five questions" section, whereas the previous form (with "can I") felt much more natural. Let me know what y'all think. 

@geduardo great spot though, thanks for the heads up. Whichever direction it goes, it definitely needs to be consistent 👍

---

Personally I'm very for having the last question be a "how", because a quantum computing newcomer reading through the questions would be given "what" it's all about in the preceding four, and then this last one is essentially an "okay cool, here are the things you can do to start learning more on your own". Essentially an answer to the _burning curiosity_ we ignited with the first four 😉 

I definitely agree though, in that "learning quantum computing" is an odd phrasing. Even classically, we'd say you either "learn programming" or "learn about computing", because of course we're not doing the computing ourselves. 

Honestly, I'm still partial to the original version---in part because I can definitely see that being a phrase that the quantum-curious might type in a search bar, and getting hits from search is something we've been notably working for